frontend/src/metabase/components/ExplicitSize/ExplicitSize.tsx

```diff
@@ -239,7 +246,19 @@ function ExplicitSize<T>({
         } else {
           return (
             <ComposedComponent
-              ref={forwardedRef}
+              ref={(el: HTMLDivElement) => {
+                if (forwardedRef) {
+                  if (typeof forwardedRef === "function") {
+                    forwardedRef(el);
+                  } else {
+                    forwardedRef.current = el;
+                  }
+                }
+
+                if (this.elementRef) {
+                  this.elementRef.current = el;
+                }
```

Comment:
we always have `this.elementRef`

```suggestion
                this.elementRef.current = el;
```